packages/garbo/src/lib.ts

```diff
@@ -1054,3 +1054,37 @@ export function aprilFoolsRufus() {
     visitUrl("questlog.php?which=7");
   }
 }
+
+type LuckyAdventure = {
+  location: Location;
+  value: () => number;
+};
+
+const luckyAdventures = [
+  {
+    location: $location`Cobb's Knob Treasury`,
+    value: () =>
+      canAdventure($location`Cobb's Knob Treasury`)
```

Comment:
i would remove canAdventure from the value func and just filter those locations out in determineBestLuckyAdventure()

Reply:
it does mean that this could possibly return nothing and you'd need to handle that unless you can guarantee some lucky location will always be available